process/Cargo.toml

```diff
@@ -14,6 +14,11 @@ libc.workspace = true
 atty.workspace = true
 errno = "0.3"
 dirs = "5.0"
+tokio = { version = "1.39", features = ["full"]}
```

Comment:
tokio with all features is too heavy for this util? Do we really need all of them or even any async runtime at all?

Reply:
@0x07C0 done.